src/app.js

```diff
@@ -94,6 +101,14 @@ class App {
   toggleLoading(value){
     $("#loadingIndicator").toggle(value);
   }
+  
+  async showStationById(poiId, poiSource) {
+    this.stationSelected({
+      id: poiId,
+      dataAdapter: poiSource,
+      charge_points: []
+    }, ">3.7", true)
```

Comment:
This doesn't yet work for POIs of other data sources (see any station in France). Also the ">3.7" (which is used for tracking), should only be set if known.
I will fix all of this (because it also requires a backend change) and try to merge it today or in the next days!